services/web/server/tests/unit/with_dbs/fast/test_access_to_studies.py

```diff
@@ -241,7 +240,7 @@ def mocks_on_projects_api(mocker) -> Dict:
     """
     state = ProjectState(
         locked=ProjectLocked(
-            value=False, owner=Owner(first_name="Speedy", last_name="Gonzalez")
+            value=False, owner=Owner(user_id=2, first_name="Speedy", last_name="Gonzalez")
```

Comment:
how come speedy gonzalez has user id 2...

Reply:
He will always be slower than the Atom Ant...